source/components/home/faq/questionList/questionList.js

```diff
@@ -167,6 +159,48 @@ class QuestionList extends Component {
                             </div>
                         </li>
                     </ul>
+                    <h2 className="faq-heading">Attendee Gas Reimbursements</h2>
+                    <ul className="column">
+                        <li>
+                            <div className="question">
+                                How do I ask for a reimbursement? 
+                            </div>
+                            <div className="answer">
+                                -The registration form has a transportation question that asks the user how they intend on getting to campus, one of the options being (Driving and will need Gas Reimbursements) <br/>
+                                -We are offering reimbursements to the first 200 attendees that sign up for one. Afterwards, reimbursements will be case by case, with RP staff having final say.
+                            </div>
+                        </li>
+                        <li>
+                            <div className="question">
+                                How much money does an attendee get?
+                            </div>
+                            <div className="answer">
+                                -$15 per attendee if location is beyond 160 miles <br/> 
+                                -$10 per attendee if location is between 60 and 160 miles <br/> 
+                                -$5 per attendee if location is within 60 miles of campus <br/> 
+                                -All distances are as the crow flies and RP staff has final say on which radius level your departure location is in <br/> 
+                                -These numbers are per attendee so they incentivize attendees to carpool
+                            </div>
+                        </li>
+                        <li>
+                            <div className="question">
+                                How will I know if I'm within the first 200 people to sign up for GR?
+                            </div>
+                            <div className="answer">
+                                Assuming you've registered, we will send you an email by Sept. 17th notifying you of your status. If you sign up after that date, we will do our best to notify you asap but cannot guarantee immediate confirmation
+                            </div>
+                        </li>
+                        <li>
+                            <div className="question">
+                                How do I collect my reimbursement?
+                            </div>
+                            <div className="answer">
+                                -We will hold a gas reimbursements booth on Fri (or Sat or both, tbd). You must have registered and explicitly noted that you need reimbursements to be given money. <br/> 
```

Comment:
> explicitly noted that you need reimbursements

(explicitly) requested a reimbursement

Try combining registered and the request for reimbursements into one phrase, or make it a causal relationship, since our official mode for any kinds requests is the registration form.